recipe/meta.yaml

```diff
@@ -104,20 +118,23 @@ outputs:
 
   - name: mamba
     version: {{ mamba_version }}
-    script: build_mamba.sh                       # [unix]
-    script: build_mamba.bat                      # [win]
+    script: build_mamba.sh                     # [unix]
+    script: bld_mamba.bat                      # [win]
     build:
       string: py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
       entry_points:
         - mamba = mamba.mamba:main
     requirements:
       build:
-        - python                              # [build_platform != target_platform]
-        - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+        - python
       host:
         - python
         - pip
         - openssl
+        - setuptools >=49.2.1
+        - pybind11 >=2.2
+        #- spdlog-fmt-embed
+        #- cli11
         - {{ pin_subpackage('libmambapy', exact=True) }}
```

Comment:
I think there is a missing host dependency `wheel`

Reply:
This section (`mamba`) is removed.